templates/parachain/README.md

```diff
@@ -11,32 +11,55 @@
 
 </div>
 
-* ⏫ This template provides a starting point to build a [parachain](https://wiki.polkadot.network/docs/learn-parachains).
+## Table of Contents
 
-* ☁️ It is based on the
+- [Intro](#intro)
+
+- [Template Structure](#template-structure)
+
+- [Getting Started](#getting-started)
+
+- [Starting a Development Chain](#starting-a-development-chain)
+
+  - [Omni Node](#omni-node)
+  - [Zombienet setup with Omni Node](#zombienet-setup-with-omni-node)
+  - [Parachain Template Node](#parachain-template-node)
+  - [Connect with the Polkadot-JS Apps Front-End](#connect-with-the-polkadot-js-apps-front-end)
+  - [Takeaways](#takeaways)
+
+- [Contributing](#contributing)
+- [Getting Help](#getting-help)
+
+## Intro
+
+- ⏫ This template provides a starting point to build a [parachain](https://wiki.polkadot.network/docs/learn-parachains).
+
+- ☁️ It is based on the
 [Cumulus](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/cumulus/index.html) framework.
 
-* 🔧 Its runtime is configured with a single custom pallet as a starting point, and a handful of ready-made pallets
+- 🔧 Its runtime is configured with a single custom pallet as a starting point, and a handful of ready-made pallets
 such as a [Balances pallet](https://paritytech.github.io/polkadot-sdk/master/pallet_balances/index.html).
 
-* 👉 Learn more about parachains [here](https://wiki.polkadot.network/docs/learn-parachains)
+- 👉 Learn more about parachains [here](https://wiki.polkadot.network/docs/learn-parachains)
 
 ## Template Structure
 
 A Polkadot SDK based project such as this one consists of:
 
-* 💿 a [Node](./node/README.md) - the binary application.
-* 🧮 the [Runtime](./runtime/README.md) - the core logic of the parachain.
-* 🎨 the [Pallets](./pallets/README.md) - from which the runtime is constructed.
+- 💿 a [Node](./node/README.md) - the binary application, not part of the project default-members list and not compiled unless
```

Comment:
Could be last on the list, as it is optional and less important here.

I think that it is enough to say that it is optional. I would move the details on how to build it into the "Parachain Template Node" section below.

Reply:
Updated here: [e6c3ab4](https://github.com/paritytech/polkadot-sdk/pull/5954/commits/e6c3ab4132a32faa3582daf29d7bfbe0915b69a5)